wpa_supplicant/wpa_cli_zephyr.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __WPA_CLI_ZEPHYR_H_
+#define __WPA_CLI_ZEPHYR_H_
+
+#include <sys/unistd.h>
+
+#include <zephyr/kernel.h>
+
+#include "includes.h"
```

Comment:
These internal includes are problematic. They should not be included in a header the app might try to use.